source/localizable/partner_badge/index.html.md.erb

```diff
@@ -0,0 +1,112 @@
+---
+title: partner_badge
+---
+
+<%= partial 'partials/page_locales' %>
+
+# Partner Badge <%= edit_link %>
+
+"Partner Badge" acts as mark of trust and a quality indicator powered by <%= settings.site_name.capitalize %>.
+
+<%= partial 'partials/toc' %>
+
+## How It Looks
+
+### Floating
+
+##### Available Themes
+
+<div class="partner_badge_themes_preview floating_badge_themes_preview">
+  <img alt="Floating Black" src="/assets/images/<%= flavor %>/partner_badge/floating_black.png">
+  <img alt="Floating Orange" src="/assets/images/<%= flavor %>/partner_badge/floating_orange.png">
+  <img alt="Floating White" src="/assets/images/<%= flavor %>/partner_badge/floating_white.png">
+</div>
+
+##### Preview
+
+<img alt="Floating Badge Preview" src="/assets/images/<%= flavor %>/partner_badge/floating_preview.gif" class="partner_badge_preview">
+
+### Embedded
+
+##### Available Themes
+
+<div class="partner_badge_themes_preview embedded_badge_themes_preview">
+  <img alt="Embedded Black" src="/assets/images/<%= flavor %>/partner_badge/embedded_black.png">
+  <img alt="Embedded White" src="/assets/images/<%= flavor %>/partner_badge/embedded_white.png">
+</div>
+
+##### Preview
+<img alt="Embedded Badge Preview" src="/assets/images/<%= flavor %>/partner_badge/embedded_preview.gif" class="partner_badge_preview">
+
+## How It Works
+
+The "<%= settings.site_name.capitalize %> Partner Badge" is displayed on your website, providing shop-related
+<%= settings.site_name.capitalize %> content upon click.
+
+- Acts as a **mark of trust** provided by <%= settings.site_name.capitalize %>, further increasing the shop's
+credibility.
+- Users will have **direct access** to shop-related <%= settings.site_name.capitalize %> content,
+such as rating, strong points, and user reviews.
+
+#### Partner Badge comes in two flavors:
+
+##### Floating
+
+Partner Badge is placed on a fixed position on the screen.
+
+##### Emebedded
+
+Partner Badge is placed on a static position of your choice and has the following dimensions: `width: 150px, height: 65px`
+
+## Integration
+
+"Partner Badge" requires to have [Analytics](/analytics/overview) integrated to your store properly.
```

Comment:
You use _Partner Badge_ with and without quotes throughout the docs.

`integrated to` should be `integrated into` or `integrated with`